projboard/tests/test_client.py

```diff
@@ -5,3 +5,19 @@
 def test_homepage(client):
     response = client.get("/")
     assert response.status_code == 200
+    template_names = set(tmpl.origin.template_name for tmpl in response.templates)
+    assert 'landing/homepage.html' in template_names
+
+
+@pytest.mark.django_db
+def test_aboutpage(client):
+    # Send a GET request to the page
+    response = client.get("/about/")
+    # Verify that the response status code is 200 (indicating a successful request)
+    assert response.status_code == 200
+    # Verify that the page contains the expected elements and content
+    assert b"<h1>Smarticle</h1>" in response.content
+    # Create a set of template names from the templates used in the response
```

Comment:
Please remove, 
This could break easily, not recommended practice, template check is enough

Reply:
Done